### PR TITLE
Add tempates for new issues and pull requests.

### LINKED ISSUE #20420: Consider using the new Issue and PR templates
GitHub just introduced templates for new issues and PRs. Read more [here](https://github.com/blog/2111-issue-and-pull-request-templates).

This could be a great way to remind folks to include `docker info` and `docker version` in their bug reports, or to add a `Signed-off-by` to their commits. @GordonTheTurtle already reminds people of these sorts of things, but it might help to have a reminder _before_ you hit the Submit button too.

I'm not well-versed enough in this project's practices to write these guidelines, but perhaps one of the maintainers could take a whack at it?

### PR DESCRIPTION
**Pull request by calavera**
I added a new `.github` directory that includes templates for new issues and pull requests.

Fixes #20420.

TODO: after this is merged
- [ ] Add same templates to https://github.com/docker/opensource

Signed-off-by: David Calavera david.calavera@gmail.com

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Please provide the following information:
+
+- What did you do?
+
+- How did you do it?
+
+- How do I see it or verify it?
+
+- A picture of a cute animal (not mandatory but encouraged)
```
Comment (MHBauer) [inline .github/PULL_REQUEST_TEMPLATE.md]: :whale:

**Comment by MHBauer** [comment]
I like both. LGTM.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (vdemeester) [inline .github/PULL_REQUEST_TEMPLATE.md]: 😍

**Comment by thaJeztah** [comment]
@calavera I was working on a template for issues, using a slightly different formatting with "HTML comments" as a trick to prevent the "boilerplating" to become visible in the issue's description. I didn't have a PR template yet, so I copied this one and applied the same "trick"; https://github.com/docker/docker/compare/master...thaJeztah:add-github-templates

I just pushed it to this repo for testing; https://github.com/thaJeztah/keyjacker/issues/new

Let me know what you think (feel free to copy) :+1:

**Comment by calavera** [comment]
That's a nice trick @thaJeztah, thanks! I've copied your templates.

**Comment by MHBauer** [comment]
the html is clever. I only hope it doesn't confuse people. They might expect it to be there. Overall I think this is still good.

**Comment by thaJeztah** [comment]
@MHBauer let's see how it goes; I hope people are smart enough to clear the template if they open a feature request as well, but not sure we can expect that to happen.

LGTM (but I'm biased :smile:)

**Comment by vdemeester** [comment]
@MHBauer @thaJeztah We can try this way and see where we end up. And iterate over this if needed :stuck_out_tongue_closed_eyes: 

LGTM too :+1: 
/cc @icecrime maybe :angel:

**Comment by thaJeztah** [comment]
I'd say; let's merge, and see if it needs tweaking \o/

**Pull request merged**